### PR TITLE
LPD-39819 Do not change the value if the input has reached the max length of the mask

### PR DESCRIPTION
**Pull request by rebsilva**
Related issues:

- https://liferay.atlassian.net/browse/LPP-56324
- https://liferay.atlassian.net/browse/LPD-39819

### CODE REVIEW

**Comment by rebsilva** [comment]
ci:test:sf

**Comment by rebsilva** [comment]
ci:test:relevant

**Comment by rebsilva** [comment]
ci:test:forms

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/Numeric/Numeric.js [hunk 1]
```diff
@@ -580,4 +582,23 @@ describe('Field Numeric', () => {
...
+		it('returns true if the input has surpassed the max length of the mask', () => {
+
+			// input smaller or with the same size as the mask should return false
+
+			expect(maxLengthExceeded('10', '90')).toBe(false);
+			expect(maxLengthExceeded('1', '90')).toBe(false);
+
+			// input longer than the size of the mask should return true
+
+			expect(maxLengthExceeded('100', '90')).toBe(true);
```
[the remark this answers is not in the document]
Comment (rebsilva) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/Numeric/Numeric.js:601]: yes, you are right! i will add this case as well, thanks!

**Comment by rebsilva** [comment]
ci:test:sf

**Review by cgmonte — APPROVED** [review]
LGTM!

**Comment by rebsilva** [comment]
ci:test:relevant

**Comment by rebsilva** [comment]
ci:forward:force

**Comment by rebsilva** [comment]
ci:forward:force

**Pull request closed without merging**